src/env.h

```diff
@@ -92,6 +92,7 @@ class ModuleWrap;
   V(decorated_private_symbol, "node:decorated")                               \
   V(npn_buffer_private_symbol, "node:npnBuffer")                              \
   V(processed_private_symbol, "node:processed")                               \
+  V(sab_lifetimepartner_symbol, "node:sharedArrayBufferLiftimePartner")       \
```

Comment:
typo? `node:sharedArrayBufferLiftimePartner -> node:sharedArrayBufferLifetimePartner`

Reply:
a) yes, thank you and b) it wasn’t even meant to be in this commit :smile: I’ve fixed it in #58, though